blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/BlueOceanUrlActionFactoryImpl.java

```diff
@@ -0,0 +1,26 @@
+package io.jenkins.blueocean;
+
+import hudson.Extension;
+import hudson.model.ModelObject;
+import io.jenkins.blueocean.rest.factory.BlueOceanUrlAction;
+import io.jenkins.blueocean.rest.factory.BlueOceanUrlActionFactory;
+import jenkins.model.Jenkins;
+
+import javax.annotation.Nonnull;
+
+/**
+ * @author Vivek Pandey
+ */
+@Extension(ordinal = -9999)
+public class BlueOceanUrlActionFactoryImpl extends BlueOceanUrlActionFactory {
```

Comment:
Agreed, it used to depend on this plugin but with this PR, we can pretty much move it inside blueocean-rest-impl.